cmd/starbridge/integrations/my_ethereum_contract.go

```diff
@@ -23,6 +27,14 @@ func IsMyContractAddress(otherAddress string) bool {
 
 var MY_CONTRACT_METHOD_HASH_SELECTORS = map[string]string{}
 
+type PaymentEvent struct {
+	ContractAddress string
+	Amount          *big.Int
+}
+
+var ethContractAddressHash = fmt.Sprintf("0x%s", common.Bytes2Hex(crypto.Keccak256([]byte("0x0000000000000000000000000000000000000000"))))
```

Comment:
Why do we hash the contract address for comparison with the address in the event?

Reply:
need it to search by that field. since we don't have that now I've removed